tracee/event_monitor_ebpf.c

```diff
@@ -1114,26 +1140,15 @@ int syscall__execve(struct pt_regs *ctx,
         return 0;
 
     context.eventid = SYS_EXECVE;
-    context.argnum = 2;
+    context.argnum = 3;
```

Comment:
what if `show_evex_args` is not set and there are just 2 args saved?

Reply:
You are right. I'll fix it